docs/docs/tutorials/set-model-parameters.md

```diff
@@ -3,5 +3,82 @@ sidebar_position: 4
+An example of specifying model parameters can be found [here](https://github.com/langchain4j/langchain4j-examples/blob/main/tutorials/src/main/java/_01_ModelParameters.java
+).
 
-Coming soon
\ No newline at end of file
+## What are Parameters in LLMs
+Depending on which model and which model provider you use, you can set a lot of parameters that will influence the model's output, speed, logging, etc.
+Typically, you will find all the parameters and their meaning on the provider's website.
+
+
+For example, OpenAI API's parameters can be found at https://platform.openai.com/docs/api-reference/chat
+and include options like
+
+| Parameter          | Description                                                                                                                                                                                                                                                                                              | Type      |
+|--------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|
+| `modelName`        | The name of the model to use (gtp-3.5-turbo, gpt-4-1106-preview, ...)                                                                                                                                                                                                                                    | `String`  |
+| `temperature`      | The sampling temperature, between 0 and 1. Higher values like 0.8 will make the output more random, while lower values like 0.2 will make it more focused and deterministic. If set to 0, the model will use log probability to automatically increase the temperature until certain thresholds are hit. | `Double`  |
+| `max_tokens`       | The maximum number of tokens that can be generated in the chat completion.                                                                                                                                                                                                                               | `Integer` |
+| `frequencyPenalty` | Number between -2.0 and 2.0. Positive values penalize new tokens based on their existing frequency in the text so far, decreasing the model's likelihood to repeat the same line verbatim.                                                                                                               | `Double`  |
+| `n`                | How many chat completion choices to generate for each input message. Note that you will be charged based on the number of generated tokens across all of the choices. Keep n as 1 to minimize costs.                                                                                                     | `Integer` |

```

Comment:
We do not allow setting `n` as our API returns single response.

Reply:
Removed it